plugin/generator.go

```diff
@@ -19,26 +19,36 @@ import (
 	"github.com/docker/docker/client"
 )
 
-var proxyServiceTasks bool
-var caddyNetworks map[string]bool
+var labelPrefixFlag string
+var proxyServiceTasksFlag bool
+var isTrue = regexp.MustCompile("^(true|True|TRUE|yes|Yes|YES|1)$")
```

Comment:
Instead of having text with different cases, we could use case insensitive regex:
https://stackoverflow.com/questions/15326421/how-do-i-do-a-case-insensitive-regular-expression-in-go
```
regexp.MustCompile("(?i)^(true|yes|1)$")
```